### PR TITLE
Adding setup.sh file

### PR DESCRIPTION
**Pull request by SaarOrus**
The setup.sh file automates the installation of necessary dependencies and configurations, reducing the setup time and minimizing potential errors.
Resloves #7

### CODE REVIEW

**Comment by talko611** [comment]
Looks great! 
After PRing my code today in order the make the automation work, could you please add on change to the file
After the line "pipenv sync --dev" (line 11) please add this line:
"cd src"
Other than that I think it would be cleaner if you edit your commits into one

**Comment by talko611** [comment]
All looks great to me!

**Comment by Ofekmn** [comment]
Looks great!

**Comment by kobihk** [comment]
@talko611 @Ofekmn I can see that you added a comment with "Looks great!" and "All looks great to me!"
but when you approve a PR we want you to add "Approve" in this way it is easier to see it.

Please add approve by:
1. Go to "File changed" tab 
2. Click on "Review changes"
3. Choose "Approve"
4. (optional) Add comment

**Review by Ofekmn — APPROVED** [review]
Looks great!

File: setup.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -ex
+# The -e option would make our script exit with an error if any command
+# fails while the -x option makes verbosely it output what it does
+
+# Install Pipenv, the -n option makes sudo fail instead of asking for a
+# password if we don�t have sufficient privileges to run it
```
Comment (kobihk) [inline setup.sh:6]: Please fix the "?" char
```suggestion
# password if we don't have sufficient privileges to run it
```

**Review by talko611 — APPROVED** [review]

**Review by kobihk — CHANGES_REQUESTED** [review]
@SaarOrus Please create an issue about this PR 
then add a description to this PR as you can see in: #2 
but please use the supported keywords as you can see in:
https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword
in this way, you make sure that when PR will merge, the relevant issue will close automatically.

**Review by rotem2022 — APPROVED** [review]

File: setup.sh [hunk 2]
```diff
@@ -3,16 +3,14 @@
...
 # Install dependencies with Pipenv
 pipenv sync --dev
-cd src
 
 # run our app. setsid, the parentheses and "&" are used to perform a "double
 # fork" so that out app stays up after the setup script finishes.
 # The app logs are redirected to the `runserver.log` file.
- (setsid pipenv run \
-       python manage.py runserver 0.0.0.0:8000 > runserver.log 2>&1 &)
+ (setsid pipenv run python manage.py runserver 0.0.0.0:8000 > runserver.log 2>&1 &)
```
Comment (kobihk) [inline setup.sh:16]: please add a new line in the end of the file

**Review by kobihk — CHANGES_REQUESTED** [review]

File: setup.sh [hunk 3]
```diff
@@ -13,4 +13,5 @@ pipenv sync --dev
 # run our app. setsid, the parentheses and "&" are used to perform a "double
 # fork" so that out app stays up after the setup script finishes.
 # The app logs are redirected to the `runserver.log` file.
- (setsid pipenv run python manage.py runserver 0.0.0.0:8000 > runserver.log 2>&1 &)
\ No newline at end of file
+ (setsid pipenv run python manage.py runserver 0.0.0.0:8000 > runserver.log 2>&1 &)
+
```
Comment (kobihk) [inline setup.sh:17]: Please remove the space from line # 17
To make sure we will not see the notification about missing a new line at the end of the file
```suggestion

```

**Review by kobihk — CHANGES_REQUESTED** [review]

**Pull request merged**